servicebindings/resolver.go

```diff
@@ -50,19 +49,17 @@ func NewResolver() *Resolver {
 //
 //  1. SERVICE_BINDING_ROOT environment variable
 //  2. CNB_BINDINGS environment variable, if above is not set
-//  3. `<platformDir>/bindings`, if both above are not set
+//  3. VCAP_SERVICES environment variable, if above is not set
+//  4. `<platformDir>/bindings`, if all above are not set
 func (r *Resolver) Resolve(typ, provider, platformDir string) ([]Binding, error) {
-	if newRoot := bindingRoot(platformDir); r.bindingRoot != newRoot {
-		r.bindingRoot = newRoot
-		bindings, err := loadBindings(r.bindingRoot)
-		if err != nil {
-			return nil, fmt.Errorf("failed to load bindings from '%s': %w", r.bindingRoot, err)
-		}
-		r.bindings = bindings
+	bindings, err := loadBindings(platformDir)
+	if err != nil {
+		// return nil, fmt.Errorf("failed to load bindings: %w", err)
```

Comment:
Can we either uncomment this line or delete it please?

Reply:
yes will delete